platform-coop-toolkit/coop-library
# feat: indicate when a subscription is required (resolves #218)

## Description

Indicates resources which require a subscription, both in card and single resource views.

## Steps to test

1. Visit Browse All and locate a resource which requires a subscription.
2. Visit the single resource view.

**Expected behavior:** The resource clearly indicates that a subscription is required (see https://github.com/platform-coop-toolkit/pinecone/issues/268).

## Additional information

Not applicable.

## Related issues

- Resolves #218.
